samples/dummy_afu/CMakeLists.txt

```diff
@@ -24,32 +24,6 @@
 ## ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 ## POSSIBILITY OF SUCH DAMAGE.
 
-set(CLI11_URL
```

Comment:
Can we keep URL and TAG variables configurable as cmake variables?

Reply:
I saw that, but they were only being used in one place.  Where else can they be used?